app/models/scenario.py

```diff
@@ -16,7 +16,18 @@ class Scenario(BaseModel):
     publish_link = Column(Text, nullable=True)
     preview_link = Column(Text, nullable=True)
     expected_solve_time = Column(Text, nullable=False, default="")
-    conclusion_data = Column(JSONB, nullable=False, default={"share_link": ""})
+    introduction_data = Column(
+        JSONB, nullable=False, default={"header_text": "Welcome to the escape room."}
+    )
+    conclusion_data = Column(
+        JSONB,
+        nullable=False,
+        default={
```

Comment:
Let's make this configurable, similar to how we do transitions, so we can change it easily once DCC gives us the default ones they want. So like:
```
default=config.get("default_data.conclusion_data")
```